olca-app/src/org/openlca/app/devtools/python/PythonEditor.java

```diff
@@ -57,7 +58,7 @@ private class Page extends FormPage {
 		private Browser browser;
 
 		public Page() {
-			super(PythonEditor.this, "PythonEditorPage", "Python");
+			super(PythonEditor.this, "PythonEditorPage", M.Python);
```

Comment:
Would there be any reason to translate _Python_?

Reply:
It is usually written with another alphabet in Arabic: https://ar.wikipedia.org/wiki/%D8%A8%D8%A7%D9%8A%D8%AB%D9%88%D9%86_(%D9%84%D8%BA%D8%A9_%D8%A8%D8%B1%D9%85%D8%AC%D8%A9).